src/garage/experiment/meta_evaluator.py

```diff
@@ -0,0 +1,102 @@
+"""Evaluator which tests Meta-RL algorithms on test environments."""
+from dowel import tabular
+import numpy as np
+
+from garage.misc.tensor_utils import discount_cumsum
+from garage.sampler import LocalSampler
+
+
+class MetaEvaluator:
+    """Evaluates Meta-RL algorithms on test environments.
+
+    Args:
+        runner (garage.experiment.LocalRunner): Runner.
+        test_task_sampler (garage.experiment.TaskSampler): Sampler for test
+            tasks. To demonstrate the effectiveness of a meta-learning method,
+            these should be different from the training tasks.
+        max_path_length (int): Maximum path length used for evaluation
+            trajectories.
+        n_test_tasks (int or None): Number of test tasks to sample each time
+            evaluation is performed. Note that tasks are sampled "without
+            replacement". If None, is set to `test_task_sampler.n_tasks`.
+        prefix (str): Prefix to use when logging. Defaults to MetaTest. For
+            example, this results in logging the key 'MetaTest/SuccessRate'.
+            If not set to `MetaTest`, it should probably be set to `MetaTrain`.
```

Comment:
can you make trivial subclasses `MetaTestEvaluator` and `MetaTrainEvaluator`?

Reply:
I don't really think that's necessary? Just pass the parameter.

Reply:
The only reason I asked was to help keep logging consistent for meta-RL algos